src/components/home/home.css

```diff
@@ -1,4 +1,41 @@
+.content-block-1 {
+    width: 100%;
+    height: 300px;
+    margin: 0;
+    padding: 30px;
+    background-color: #f2f2f2;
+}
+
+.home-h1 {
+    color: #5E5D5D;
+    width: 85%;
+    max-width: 1000px;
+    margin: auto;
+    text-align: center;
+    /* padding-top: 5%; */
```

Comment:
Nit: A PR should never have commented out code. If you don't need it, delete it. If you aren't sure yet, then the code probably isn't ready for review/PR